lua/nvim-web-devicons.lua

```diff
@@ -1453,7 +1453,7 @@ local icons_by_file_extension = {
     name = "Vim",
   },
   ["vue"] = {
-    icon = "",
+    icon = "﵂",
```

Comment:
Unfortunately this is a nerd font v2 icon; we need a v3 icon.

See #264

You'll need to run nerdfix.